tests/test_cli.py

```diff
@@ -169,3 +169,44 @@ def test_config_passed_down_to_command_children(
             "params": {"time": 5},
         }
     }
+
+
+def test_invalid_stomp_config_for_listener(runner: CliRunner):
+    result = runner.invoke(main, ["controller", "listen"])
+    assert (
+        isinstance(result.exception, RuntimeError)
+        and str(result.exception) == "Message bus needs to be configured"
+    )
+
+
+def test_cannot_run_plans_without_stomp_config(runner: CliRunner):
+    with patch("uvicorn.run", side_effect=None):
+        result = runner.invoke(main, ["serve"])
+    assert result.exit_code == 0
```

Comment:
As discussed, we want a server that is configured with the same communication methods as the client, so that behaviour when other communication methods are added is clear without editing the tests.